packages/domain-picker/src/domain-picker/use-your-domain-item.tsx

```diff
@@ -8,14 +8,22 @@ import { ArrowButton } from '@automattic/onboarding';
 /**
  * Internal dependencies
  */
+import WrappingComponent from './suggestion-item-wrapper';
 
 interface Props {
 	onClick: () => void;
 }
 
 const UseYourDomainItem: React.FunctionComponent< Props > = ( { onClick } ) => {
+	const useYourOwnDomainItemRef = React.createRef< any >();
+
 	return (
-		<label className="domain-picker__suggestion-item contains-link">
+		<WrappingComponent
+			type="button"
+			className="domain-picker__suggestion-item type-link"
+			ref={ useYourOwnDomainItemRef }
```

Comment:
Forgot to mention — we can probably delete this line (and the `useYourOwnDomainItemRef` variable declaration)